.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
-- repo: git@github.com:Yelp/detect-secrets
+- repo: https://github.com/Yelp/detect-secrets.git
```

Comment:
@cagiti is this valid? I don't use ssh for git at all so this seems more universal, as everyone has http these days.